### DISCUSSION TITLE
Is the `/api/v1/chart=<chart-id>` correct?

### DISCUSSION THREAD

**Discussion by 0xNima** [netdata/netdata#16390] [Q&A]
We've setup Netdata on numerous servers and monitor them by tv.html. But after recent updates, some charts are not displayed. I found that the potential problem is [this line](https://github.com/netdata/netdata/blob/826001011f4992c39b9860da7d14501ff2ff97b2/web/api/formatters/rrdset2json.c#L37).
Because the chart's data is available on `/api/v1/chart?chart=<chart-id>` not `/api/v1/chart=<chart-id>`.

**Comment by ilyam8** [comment]
Hi, @0xNima. It is a bug, fixed in https://github.com/netdata/netdata/pull/16389.
